### PR TITLE
Bring back keyboard shortcuts to help modal on non-mobile

### PR DESCRIPTION
**Pull request by mejo-**

### CODE REVIEW

File: src/components/HelpModal.vue [hunk 1]
```diff
@@ -180,7 +180,7 @@
 
 <script>
 import { NcModal, Tooltip } from '@nextcloud/vue'
-import { isMobilePlatform } from './../helpers/platform.js'
```
[the remark this answers is not in the document]
Comment (mejo-) [reply src/components/HelpModal.vue:183]: Well, the problem here was that `isMobilePlatform` is a method, not a computed property. After changing the `v-if` statements to `!isMobilePlatform()` it worked as well. Updated the PR accordingly now.

I also went with caching the result of method `isMobilePlatform()` as computed property, given that it doesn't change within one browser anyway.

**Review by juliushaertl — APPROVED** [review]

**Comment by mejo-** [comment]
/backport 514930907b2b9b018c4f8bb3ab6f2fb654b20723,4dae22a6fcea13cb25ee997335fdcf5005373f42 to stable27

**Comment by mejo-** [comment]
/backport 514930907b2b9b018c4f8bb3ab6f2fb654b20723,4dae22a6fcea13cb25ee997335fdcf5005373f42 to stable26

**Pull request merged**